app/views.py

```diff
@@ -78,6 +78,80 @@ def admin():
     return render_template("admin.html", vehicles=vehicles, services=services)
 
 
+@app.route("/admin/services")
+def admin_services_management():
+    services = services_model.all()
+    return render_template("admin/services/index.html", services=services)
+
+
+@app.route("/admin/services/adding", methods=["GET", "POST"])
+def admin_services_adding():
+    if request.method == "POST":
+        images = None
+        if request.files.getlist("images"):
+            images = util.upload_files(
+                request.files.getlist("images"), app.config["IMAGES_DIR_PATH"]
+            )
+
+        price = [price.split(",") for price in request.form.getlist("price")]
+        services_model.insert(
+            name=request.form["name"],
+            price=price,
+            description=request.form["description"],
+            images=images,
+        )
+        flash('Service has been created')
+        return jsonify(data="OK")
+    return render_template("admin/services/upsert.html", mode="add")
+
+
+@app.route("/admin/services/<id>/editing", methods=["GET", "POST"])
```

Comment:
same note as above, prefer to have the method name as `edit` instead of `editing`

Reply:
I'll update it